assets/js/modules/pagespeed-insights/datastore/report.test.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * modules/pagespeed-insights data store: report tests.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import API from 'googlesitekit-api';
+import { STORE_NAME } from './index';
+import {
+	createTestRegistry,
+	muteConsole,
+	subscribeUntil,
+	unsubscribeFromAll,
+} from 'tests/js/utils';
+import * as fixtures from './__fixtures__';
+
+describe( 'modules/pagespeed-insights report', () => {
+	let registry;
+
+	beforeAll( () => {
+		API.setUsingCache( false );
+	} );
+
+	beforeEach( () => {
+		registry = createTestRegistry();
+	} );
+
+	afterAll( () => {
+		API.setUsingCache( true );
+	} );
+
+	afterEach( () => {
+		unsubscribeFromAll( registry );
+	} );
+
+	describe( 'actions', () => {
```

Comment:
We shouldn't leave empty describe blocks, so normally I'd say to remove this… but in this case I'd say it's a good place to add a test for the `fetchReport()` action that wasn't returning the correct data above 😄

Reply:
Makes sense, added a test.